studio/localStores/sqlEditor/SqlEditorStore.js

```diff
@@ -279,10 +298,15 @@ class SqlEditorStore {
   }
 
   closeTab(id) {
-    this.tabs = this.tabs.filter((x) => x.id !== id)
+    const tabs = this.tabs.filter((x) => x.id !== id)
```

Comment:
do we not need to update `this.tabs`?

Reply:
`this.tabs` gets updated when you call `sqlEditorStore.loadTabs(sqlEditorStore.tabsFromContentStore(contentStore,user?.id))`. I figured since `loadTabs` is already handling removing a tab, `closeTab` should just be responsible for switching away from a tab and not removing it from the store. Happy to reconsider if you think it's better the other way!

Reply:
oh yup - that's fair, but just to check those closeTab here call `loadTabs` to refresh the tabs?

Reply:
They're separate now, so you can close a tab without removing it from the list. I'm calling `loadTabs` after my call to `closeTab`. Do you think it's better to call it from inside `closeTab`? Personally, I like the separation.